<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projekt_kat0013Class</class>
 <widget class="QMainWindow" name="projekt_kat0013Class">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>projekt_kat0013</string>
  </property>
  <property name="iconSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="ptNumbers">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>700</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPlainTextEdit" name="pte">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>700</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSoubor">
    <property name="title">
     <string>Soubor</string>
    </property>
    <addaction name="actionNov"/>
    <addaction name="actionOtev_t"/>
    <addaction name="actionUlo_it"/>
    <addaction name="actionUlo_it_jako"/>
    <addaction name="separator"/>
    <addaction name="actionVzhled_str_nky"/>
    <addaction name="actionTisk"/>
    <addaction name="separator"/>
    <addaction name="actionKonec"/>
   </widget>
   <widget class="QMenu" name="menu_pravy">
    <property name="title">
     <string>Úpravy</string>
    </property>
    <addaction name="actionZp_t"/>
    <addaction name="actionVyjmout"/>
    <addaction name="actionKop_rovat"/>
    <addaction name="actionVlo_it"/>
    <addaction name="actionOstranit"/>
    <addaction name="separator"/>
    <addaction name="actionNaj_t"/>
    <addaction name="actionNaj_t_dal"/>
    <addaction name="actionNahradit"/>
    <addaction name="actionP_ej"/>
    <addaction name="separator"/>
    <addaction name="actionVybrat_v_e"/>
    <addaction name="action_as_a_datum"/>
   </widget>
   <widget class="QMenu" name="menuForm_t">
    <property name="title">
     <string>Formát</string>
    </property>
    <addaction name="actionZalamov_n_dk"/>
    <addaction name="actionP_smo"/>
   </widget>
   <widget class="QMenu" name="menuZobrazen">
    <property name="title">
     <string>Zobrazení</string>
    </property>
    <addaction name="actionStavov_dek"/>
   </widget>
   <widget class="QMenu" name="menuN_pov_da">
    <property name="title">
     <string>Nápověda</string>
    </property>
    <addaction name="actionZobrazit_n_pov_du"/>
    <addaction name="separator"/>
    <addaction name="actionO_programu"/>
   </widget>
   <addaction name="menuSoubor"/>
   <addaction name="menu_pravy"/>
   <addaction name="menuForm_t"/>
   <addaction name="menuZobrazen"/>
   <addaction name="menuN_pov_da"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="actionLock"/>
   <addaction name="actionSetting"/>
   <addaction name="actionBack"/>
   <addaction name="actionRepeat"/>
   <addaction name="separator"/>
   <addaction name="actionLeftalign"/>
   <addaction name="actionCenteralign"/>
   <addaction name="actionRightalign"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionZoomout"/>
   <addaction name="actionZoomin"/>
   <addaction name="actionFce"/>
   <addaction name="actionImg"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalics"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNov">
   <property name="text">
    <string>Nový</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionOtev_t">
   <property name="text">
    <string>Otevřít</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionUlo_it">
   <property name="text">
    <string>Uložit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionVzhled_str_nky">
   <property name="text">
    <string>Vzhled stránky</string>
   </property>
  </action>
  <action name="actionTisk">
   <property name="text">
    <string>Tisk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionKonec">
   <property name="text">
    <string>Konec</string>
   </property>
  </action>
  <action name="actionZp_t">
   <property name="text">
    <string>Zpět</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionVyjmout">
   <property name="text">
    <string>Vyjmout</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionKop_rovat">
   <property name="text">
    <string>Kopírovat</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionVlo_it">
   <property name="text">
    <string>Vložit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionOstranit">
   <property name="text">
    <string>Ostranit</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionNaj_t">
   <property name="text">
    <string>Najít</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionNaj_t_dal">
   <property name="text">
    <string>Najít další</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionNahradit">
   <property name="text">
    <string>Nahradit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionVybrat_v_e">
   <property name="text">
    <string>Vybrat vše</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_as_a_datum">
   <property name="text">
    <string>Čas a datum</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionUlo_it_jako">
   <property name="text">
    <string>Uložit jako</string>
   </property>
  </action>
  <action name="actionP_ej">
   <property name="text">
    <string>Přejí</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionZalamov_n_dk">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zalamování řádků</string>
   </property>
  </action>
  <action name="actionP_smo">
   <property name="text">
    <string>Písmo...</string>
   </property>
  </action>
  <action name="actionStavov_dek">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stavový řádek</string>
   </property>
  </action>
  <action name="actionZobrazit_n_pov_du">
   <property name="text">
    <string>Zobrazit nápovědu</string>
   </property>
  </action>
  <action name="actionO_programu">
   <property name="text">
    <string>O programu</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset>
     <normaloff>../icons/load.png</normaloff>../icons/load.png</iconset>
   </property>
   <property name="text">
    <string>load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>../icons/save.png</normaloff>../icons/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actionLock">
   <property name="icon">
    <iconset>
     <normaloff>../icons/lock.png</normaloff>../icons/lock.png</iconset>
   </property>
   <property name="text">
    <string>lock</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset>
     <normaloff>../icons/setting.png</normaloff>../icons/setting.png</iconset>
   </property>
   <property name="text">
    <string>setting</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset>
     <normaloff>../icons/undo.png</normaloff>../icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </action>
  <action name="actionRepeat">
   <property name="icon">
    <iconset>
     <normaloff>../icons/repeat.png</normaloff>../icons/repeat.png</iconset>
   </property>
   <property name="text">
    <string>repeat</string>
   </property>
  </action>
  <action name="actionLeftalign">
   <property name="icon">
    <iconset>
     <normaloff>../icons/left_align.png</normaloff>../icons/left_align.png</iconset>
   </property>
   <property name="text">
    <string>leftalign</string>
   </property>
  </action>
  <action name="actionCenteralign">
   <property name="icon">
    <iconset>
     <normaloff>../icons/justify_align.png</normaloff>../icons/justify_align.png</iconset>
   </property>
   <property name="text">
    <string>centeralign</string>
   </property>
  </action>
  <action name="actionRightalign">
   <property name="icon">
    <iconset>
     <normaloff>../icons/right_align.png</normaloff>../icons/right_align.png</iconset>
   </property>
   <property name="text">
    <string>rightalign</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset>
     <normaloff>../icons/bold.png</normaloff>../icons/bold.png</iconset>
   </property>
   <property name="text">
    <string>bold</string>
   </property>
  </action>
  <action name="actionItalics">
   <property name="icon">
    <iconset>
     <normaloff>../icons/italics.png</normaloff>../icons/italics.png</iconset>
   </property>
   <property name="text">
    <string>italics</string>
   </property>
  </action>
  <action name="actionActionCbx">
   <property name="text">
    <string>actionCbx</string>
   </property>
  </action>
  <action name="actionZoomin">
   <property name="icon">
    <iconset>
     <normaloff>../icons/zoomin.png</normaloff>../icons/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>zoomin</string>
   </property>
  </action>
  <action name="actionZoomout">
   <property name="icon">
    <iconset>
     <normaloff>../icons/zoomout.png</normaloff>../icons/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>zoomout</string>
   </property>
  </action>
  <action name="actionFce">
   <property name="icon">
    <iconset>
     <normaloff>../icons/function.gif</normaloff>../icons/function.gif</iconset>
   </property>
   <property name="text">
    <string>fce</string>
   </property>
  </action>
  <action name="actionImg">
   <property name="icon">
    <iconset>
     <normaloff>../icons/img.png</normaloff>../icons/img.png</iconset>
   </property>
   <property name="text">
    <string>img</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>../icons/print.png</normaloff>../icons/print.png</iconset>
   </property>
   <property name="text">
    <string>print</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>../icons/copy.png</normaloff>../icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>../icons/cut.png</normaloff>../icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>cut</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="projekt_kat0013.qrc"/>
 </resources>
 <connections/>
</ui>
